ci-operator/config/openshift/etcd/OWNERS

```diff
@@ -0,0 +1,24 @@
+# DO NOT EDIT; this file is auto-generated using tools/populate-owners.
+# from https://github.com/openshift/etcd/blob/ea0cf681c7f0278eff72846a6bfcd017cdfc83bf/OWNERS
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
```

Comment:
Unfortunately this is not who can approve these changes.  @stevekuznetsov how do we exclude autogenerated owners / override?  etcd has an upstream, we have a fork, the fork needs tighter control